META.md

```diff
@@ -1,6 +1,6 @@
 # Generating changelog
 
-Use: https://github.com/skywinder/Github-Changelog-Generator
+Use: https://github.com/skywinder/GitHub-Changelog-Generator
```

Comment:
Actually, the correct path for this is all lowercase: https://github.com/skywinder/gitlab-changelog-generator